src/management/provision.ts

```diff
@@ -0,0 +1,189 @@
+import { Client } from "../apiv2";
```

Comment:
My 2c is there is enough logic to create a `src/management/provision` folder with modular files in it.

Glancing through the PR. We can organize logics into `client.ts`, `setup_apps.ts`, etc.